mora/service/cpr.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2017-2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+'''CPR
+-----------
+
+This section describes functionality for retrieving information about people
+based on CPR..
+'''
+
+import flask
+
+from mora.integrations.serviceplatformen import get_citizen
+from mora.service import keys
+
+blueprint = flask.Blueprint('cpr', __name__, static_url_path='',
+                            url_prefix='/service')
+
+
+@blueprint.route('/e/cpr_lookup/<string:cpr>/')
+def search_cpr(cpr):
+    """
+    Search for CPR in Serviceplatformen
+
+    :param cpr: The CPR no. of a person to be searched
+    """
+
+    if len(cpr) != 10:
+        raise ValueError('CPR should be 10 characters long')
```

Comment:
Although this occurs many other places, I'd say the proper nomenclature is “CPR number”. Or _personal identification number_ if we want to be accurate and annoying 😉